From 79b9e4338f803d79449e53a40b1ecc0a5a5889e4 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Tue, 26 Oct 2021 08:52:17 +0200
Subject: [PATCH] waffle: do not make core protocol into the library

None of the consumers (which is just piglit) use it, and
this avoids host contamination from pkg-config suggesting
wayland.xml from the host.

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Alexander Kanavin <alex@linutronix.de>
---
 src/waffle/meson.build | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/src/waffle/meson.build b/src/waffle/meson.build
index 1cc99f6..022745a 100644
--- a/src/waffle/meson.build
+++ b/src/waffle/meson.build
@@ -89,12 +89,6 @@ if build_surfaceless
 endif
 
 if build_wayland
-  wl_core_proto_c = custom_target(
-    'wl-core-proto.c',
-    input: wayland_core_xml,
-    output: 'wl-core-proto.c',
-    command: [prog_wayland_scanner, 'private-code', '@INPUT@', '@OUTPUT@'],
-  )
   wl_xdg_shell_proto_c = custom_target(
     'wl-xdg-shell-proto.c',
     input: wayland_xdg_shell_xml,
@@ -115,7 +109,6 @@ if build_wayland
     'wayland/wayland_wrapper.c',
   )
   files_libwaffle += [
-    wl_core_proto_c,
     wl_xdg_shell_proto_c,
     wl_xdg_shell_proto_h,
   ]
